.github/workflows/release.yml

```diff
@@ -58,7 +98,7 @@ jobs:
           git push
           git push --tags
           noDotVersion="${current_version//.}"
-          gh release create "release/${current_version}" --title "${current_version}" --notes "# Download
+          gh release create "release/${current_version}" ./cli/*.tar.gz --title "${current_version}" --notes "# Download
           https://repo1.maven.org/maven2/com/devonfw/tools/ide/devonfw-ide-cli/${current_version}/devonfw-ide-cli-${current_version}.tar.gz
```

Comment:
That link seems to be outdated. I guess it would be like this:
https://repo1.maven.org/maven2/com/devonfw/tools/IDEasy/ide-cli/${current_version}/ide-cli-${current_version}.tar.gz

Reply:
Done